beetsplug/play.py

```diff
@@ -129,6 +134,15 @@ def _play_command(self, lib, opts, args):
         open_args = self._playlist_or_paths(paths)
         command_str = self._command_str(opts.args)
 
+        if PLS_MARKER in command_str:
+            if not config['play']['raw']:
+                ui.print_(ui.colorize('text_warning', 'DEBUG: ' + command_str))
```

Comment:
Seems like these should probably be proper debug logs… like `log.debug(...)`?

Reply:
done. totally overlooked this.